Libraries/LibELF/DynamicLoader.h

```diff
@@ -144,6 +145,8 @@ class DynamicLoader : public RefCounted<DynamicLoader> {
 
     size_t m_tls_offset { 0 };
     size_t m_tls_size { 0 };
+
+    Vector<DynamicObject::Relocation> m_lazy_relocations;
```

Comment:
Maybe a nitpick, but i think that this name could be confusing readers with lazy binding in PLT.
Perhaps `m_unresolved_relocations` or `m_late_relocations` would be better names?